client/post-editor/editor-image-selector/preview.jsx

```diff
@@ -0,0 +1,223 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { defer, isEqual, uniq } from 'lodash';
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ImagePreloader from 'components/image-preloader';
+import { localize } from 'i18n-calypso';
+import MediaActions from 'lib/media/actions';
+import MediaStore from 'lib/media/store';
+import Spinner from 'components/spinner';
+import { url } from 'lib/media/utils';
+
+export class EditorImageSelectorPreview extends React.Component {
+	static displayName = 'EditorImageSelectorPreview';
```

Comment:
I don't think this is required anymore, also why not just import `Component` directly and extend it here?

Reply:
Updated both.